app/assets/stylesheets/_entries.scss

```diff
@@ -7,7 +7,7 @@
     @include pad(5%)
   }
 
-  #new_entry, .edit_entry {
+  #new_hour, .edit_hour {
```

Comment:
Avoid using id selectors<br>Selector `new_hour` should be written in lowercase with hyphens<br>Selector `edit_hour` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line